### PR TITLE
PD-2384 Some UI updates per the ticket notes

### PR DESCRIPTION
**Pull request by misterjonscott**
Fixed autofocus error introduced into QA
Added button highlight capability (global)
Changed the SearchInput (partner report > URL, Source) to use the dropdown class
Fixed broken proptype in Sortable List
Made heading text smaller in preview table
Added report name to export report page
Added Export instructions

### CODE REVIEW

**Comment by aspidites** [comment]
Merge conflicts.

**Comment by darrint** [comment]
The rendering seems slightly off:

2016-05-12-141021_329x207_scrot

The bullet points look bigger darker and bolder. Is that ok?

**Comment by aspidites** [comment]
The dropdown class is rendering off-center for me:
2016-05-12-141226_533x240_scrot

There also seems to be some extra padding in the select entries:
2016-05-12-141740_501x391_scrot

File: static/custom.css [hunk 1]
```diff
@@ -3456,7 +3460,7 @@ fieldset:nth-child(n+2) {
 }
 
 .row .col-md-8 ul {
-    margin: 0;
+    margin: 0 0 0 20px;
```
Comment (tdphillips) [inline static/custom.css]: This is affecting the placement of search fields.

Comment (misterjonscott) [inline static/custom.css]: Ugh.  Got it.

**Pull request merged**